code/web/src/modules/user/MyAccount.js

```diff
@@ -0,0 +1,273 @@
+          onClick={this.changeEmailEditMode}
+        >
+          ✔️
+        </Button>
+      </div>
+    )
+  }
+
+  renderEmailDefaultView = () => {
+    return (
+      <div style={{ display: 'flex', alignItems: 'center' }}>
+        <p style={{ color: black }}>{this.state.email}</p>
+        <Button
+          type="submit"
+          theme="secondary"
+          style={{ display: 'flex', alignItems: 'center', height: '1.9em', marginLeft: '2em' }}
+          onClick={this.changeEmailEditMode}
+        >
+          <img src={`${ APP_URL }/images/icons8-edit-24.png`} alt="Edit" />
+        </Button>
+      </div>
+    )
+  }
+
+  changeAddressEditMode = () => {
+    event.preventDefault()
+
+    this.setState({
+      addressEditMode: !this.state.addressEditMode
+    })
+  }
+
+  renderAddressEditView = () => {
+    return (
+      <div style={{ display: 'flex', alignItems: 'center' }}>
+        <Input
+          type='text'
+          name='address'
+          value={this.state.address}
+          onChange={this.handleChange}
+        />
+        <Button
+          type="submit"
+          theme="secondary"
+          style={{ display: 'flex', alignItems: 'center', height: '1.9em', marginLeft: '2em' }}
+          onClick={this.changeAddressEditMode}
+        >
+          ✔️
+        </Button>
+      </div>
+    )
+  }
+
+  renderAddressDefaultView = () => {
+    return (
+      <div style={{ display: 'flex', alignItems: 'center' }}>
+        <p style={{ color: black }}>{this.state.address}</p>
+        <Button
+          type="submit"
+          theme="secondary"
+          style={{ display: 'flex', alignItems: 'center', height: '1.9em', marginLeft: '2em' }}
+          onClick={this.changeAddressEditMode}
+        >
+          <img src={`${ APP_URL }/images/icons8-edit-24.png`} alt="Edit" />
+        </Button>
+      </div>
+    )
+  }
+
+  render() {
+    return(
+      <>
+        <SubNav />
+
+        <Grid>
+          <GridCell style={{ padding: '2em', textAlign: 'center' }}>
+            <H4 style={{ marginBottom: '0.5em' }}>User's Name</H4>
+
+            <form style={{ display: 'flex', alignItems: 'center', justifyContent: 'center' }}>
+
+              <section style={{ display: 'flex', flexDirection: 'column', alignItems: 'center', justifyContent: 'space-around', width: '40%', height: '60vh' }}>
+
+                {/* Profile Image */}
+                <div style={{ width: '200px', height: '200px', position: 'relative', overflow: 'hidden', borderRadius: '50%' }}>
+                  <img
+                    src='https://hips.hearstapps.com/countryliving.cdnds.net/17/47/2048x1365/gallery-1511194376-cavachon-puppy-christmas.jpg?resize=768:*'
+                    alt='profile image'
+                    style={{ display: 'inline', margin: '0 auto', marginLeft: '-25%', height: '100%', width: 'auto' }}
+                  />
+                </div>
+                <File
```

Comment:
If we wanted to match this to the button styling of the other 'edit' buttons here, we could probably pass an inline style that would overwrite the component one, but I think it looks nice either way.